### ISSUE TITLE
load from io streams

### ISSUE THREAD

**Issue by danodonovan** [avakar/pytoml#23]
I just encountered an issue when trying to `pytoml.load` from a python `io.StringIO` (or other stream). PR #22 is a quick fix - NB no extra tests added.

**Comment by danodonovan** [comment]
For example

```python
>>> import io
>>> import pytoml
>>> pytoml.load(io.StringIO(u"this_is = 'a_test'"))
```

> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "lib/python2.7/site-packages/pytoml/parser.py", line 10, in load
>     return loads(fin.read(), translate=translate, filename=fin.name)
> AttributeError: '_io.StringIO' object has no attribute 'name'

**Comment by avakar** [comment]
Sorry for the late response, I completely missed out on the newer issues. I'll look into the problem ASAP.

**Comment by avakar** [comment]
Fixed in 0.1.12, upgrade with pip install --upgrade pytoml. Thanks!
